ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -627,6 +627,30 @@
 {%             endfor %}
 {%         endif %}
 {%     endif %}
+{%     if router_bgp.address_family_ipv4_path_selection is arista.avd.defined %}
+
+##### Path-Selection Neighbors
+
+| Neighbor | Activate | Route-map In | Route-map Out |
+| -------- | -------- | ------------ | ------------- |
+{%         for neighbor in router_bgp.address_family_ipv4_path_selection.neighbors | arista.avd.natural_sort('ip_address') %}
+{%             set route_map_in = neighbor.route_map_in | arista.avd.default("-") %}
+{%             set route_map_out = neighbor.route_map_out | arista.avd.default("-") %}
+| {{ neighbor.ip_address }} | {{ neighbor.activate | arista.avd.default(false) }} | {{ route_map_in }} | {{ route_map_out }} |
+{%         endfor %}
+{%     endif %}
+{%     if router_bgp.address_family_ipv4_path_selection.peer_groups is arista.avd.defined %}
+
+##### Path-Selection Peer Groups
+
+| Peer Group | Activate | Route-map In | Route-map Out |
+| ---------- | -------- | ------------ | ------------- |
+{%         for peer_group in router_bgp.address_family_ipv4_path_selection.peer_groups | arista.avd.natural_sort('name') %}
+{%             set route_map_in = peer_group.route_map_in | arista.avd.default("-") %}
+{%             set route_map_out = peer_group.route_map_out | arista.avd.default("-") %}
+| {{ peer_group.name }} | {{ peer_group.activate | arista.avd.default(false) }} | {{ route_map_in }} | {{ route_map_out }} |
```

Comment:
The input model does not offer route-maps for peer-groups. Not sure if EOS would support it on this AF.